tendermint/tests/rpc.rs

```diff
@@ -47,29 +47,28 @@ mod endpoints {
 
         assert_eq!(header.version.block, 10);
         assert_eq!(header.chain_id.as_str(), EXAMPLE_CHAIN);
-        assert_eq!(header.height.value(), 15);
-        assert_eq!(header.num_txs, 2);
-
-        assert_eq!(data.iter().len(), 2);
+        assert_eq!(header.height.value(), 10);
+        assert_eq!(data.iter().len(), 0);
         assert_eq!(evidence.iter().len(), 0);
-        assert_eq!(last_commit.unwrap().precommits.len(), 65);
+        assert_eq!(last_commit.unwrap().signatures.len(), 1);
     }
 
-    #[test]
-    fn block_empty_block_id() {
-        let response =
-            endpoint::block::Response::from_string(&read_json_fixture("block_empty_block_id"))
-                .unwrap();
-
-        let tendermint::Block { last_commit, .. } = response.block;
-
-        assert_eq!(last_commit.as_ref().unwrap().precommits.len(), 2);
-        assert!(last_commit.unwrap().precommits[0]
-            .as_ref()
-            .unwrap()
-            .block_id
-            .is_none());
-    }
+    // TODO: Update this test and its json file
+    // #[test]
+    // fn block_empty_block_id() {
+    //     let response =
+    //         endpoint::block::Response::from_string(&read_json_fixture("block_empty_block_id"))
+    //             .unwrap();
+    //
+    //     let tendermint::Block { last_commit, .. } = response.block;
+    //
+    //     assert_eq!(last_commit.as_ref().unwrap().precommits.len(), 2);
+    //     assert!(last_commit.unwrap().precommits[0]
+    //         .as_ref()
+    //         .unwrap()
+    //         .block_id
+    //         .is_none());
+    // }
```

Comment:
Will this be added back in this PR or later?

Reply:
Nice find, I missed that. I'll let Shivani decide. She's working on more tests, that's why I didn't wait with opening this PR. I expect more tests will come either into this PR (bloating it even more) or in new PRs after this has been merged (which seems to cut our work better for us).

Reply:
Yes @greg-szabo, I'll let these test-related things come in a separate PR.